### DISCUSSION TITLE
Proposal: to make the texts of the "Name Teilnehmer*in" field changeable in the pretix admin panel

### DISCUSSION THREAD

**Discussion by Wikinaut** [pretix/pretix#4528] [Q&A] [answered]
![grafik](https://github.com/user-attachments/assets/e616cb11-8bc9-47c4-b1c9-4a67c5cd842d)

As far as I currently understand, this field text (here: ```Name Teilnehmer*in```) is fixed and translated in https://translate.pretix.eu/ .

For certain projects (like my current), I received the request to adapt (change) the text, so I post the idea of having it changeable in the pretix configuration GUI (admin panel).

Also the default value (here: ```Name```) should perhaps become changeable.

Edited:
The requester wanted to have ```Name or Nickname``` as amended field texts for their events.

**Comment by pc-coholic** [comment, marked as answer]
We've discussed this before - and it's a hard "no" on that one for us, sorry.

If you're self-hosted, you can work around this by providing your own translations - but adding a bunch more configuration boxes for something like this, is something we actively decided against.

**Comment by Wikinaut** [comment]
Ok. Just as a follow-up, I edited:
"The requester wanted to have ```Name or Nickname``` as amended field texts for their events."

**Reply by pc-coholic** [reply]
Perhaps a workaround: Set the name format only to "Ask for name" and use the "Attendee data explanation" section.

Or: Disable the system-fields and create a custom question.

**Reply by Wikinaut** [reply]
> (1) Perhaps a workaround: Set the name format only to "Ask for name" and use the "Attendee data explanation" section.
> (2) Or: Disable the system-fields and create a custom question.
> 
> 

(1) works for me, is a help at least. Thanks.

(2) If I use this method, how could then the important attendee-**name** information be used in the order and billing process, is this possible by your proposed method with a custom question?

**Reply by pc-coholic** [reply]
It depends...

Using a custom question, the name of the attendee will not make it into the billing address - but the system fields wouldn't do that either. For that, you would need to specifically use the invoice address-feature, since the attendee name and questions are always tied to an order position and not an order as a whole.

You can however (in the advanced settings of a question) designate a question-answer to be printed on the invoice in the itemized overview.

You cannot however use the answer to a question in emails.
